src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/endpoint/CaseEndpointInvalidateCaseTest.java

```diff
@@ -39,6 +41,8 @@
 public final class CaseEndpointInvalidateCaseTest {
 
   @Mock private CaseService caseService;
+  @Mock private CaseQueryRequestDTO caseQueryRequestDTO;
```

Comment:
these 2 mocks are not used. Please revert this class to its original form pre-PR, since there are no changes here apart from commented out code and unused variables.

Reply:
done